rhizome/postgres/bin/configure

```diff
@@ -98,6 +98,36 @@ standby2replication   #{identity}             ubi_replication
 PG_IDENT
 safe_write_to_file("/etc/postgresql/#{v}/main/pg_ident.conf", pg_ident_entries)
 
+# Configure system hugepages
+meminfo = File.read("/proc/meminfo")
+hugepage_size_kib = Integer(meminfo[/^Hugepagesize:\s*(\d+)\s*kB/, 1], 10)
+memory_total_kib = Integer(meminfo[/^MemTotal:\s*(\d+)\s*kB/, 1], 10)
+
+target_hugepages_size_kib = memory_total_kib / 4
+target_hugepages = target_hugepages_size_kib / hugepage_size_kib
+
+r "echo 'vm.nr_hugepages = #{target_hugepages}' | sudo tee /etc/sysctl.d/10-hugepages.conf"
+r "sudo sysctl --system"
```

Comment:
Before applying sysctl, you might find the odds of getting the contiguous memory improve if you do this:

```
sync
echo 3 | sudo tee /proc/sys/vm/drop_caches
echo 1 | sudo tee /proc/sys/vm/compact_memory
```

You don't need to do it now, maybe in practice you always have enough contiguous memory.